x/mint/keeper/genesis.go

```diff
@@ -5,13 +5,19 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// InitGenesis new mint genesis
+// InitGenesis initializes the x/mint store with data from the genesis state.
 func (keeper Keeper) InitGenesis(ctx sdk.Context, ak types.AccountKeeper, data *types.GenesisState) {
 	keeper.SetMinter(ctx, data.Minter)
+	// override the genesis time with the actual genesis time supplied in `InitChain`
+	blockTime := ctx.BlockTime()
+	gt := types.GenesisTime{
+		GenesisTime: &blockTime,
+	}
+	keeper.SetGenesisTime(ctx, gt)
```

Comment:
[Non blocking][To understand more]
This is still setting the genesis time in state, no?

Reply:
Yep this is setting the genesis time in the state. Notably it doesn't use the mint module's genesis state `genesis_time` field because that was removed. It uses the root level genesis time from genesis.json